### PR TITLE
New command, 'Hungry' 

### PR DESCRIPTION
**Pull request by vikingcode**
This is literally the recipe that inspired the name and was used for the photo on http://code52.org/pretzel/

### CODE REVIEW

**Comment by JakeGinnivan** [comment]
Haha, awesome :shipit:

**Comment by laedit** [comment]
Excellent :smiley:

File: src/Pretzel/Commands/HungryCommand.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Eat the pretzels warm, or reheat them in an oven or microwave.";
+
+        public void Execute(IEnumerable<string> arguments)
+        {
+            Tracing.Info(recipe);
+        }
+
+        public void WriteHelp(TextWriter writer)
+        {
+            writer.Write(recipe);
```
Comment (laedit) [inline src/Pretzel/Commands/HungryCommand.cs]: This is a little bit annoying when one try to understand how to use pretzel and show help...

Comment (vikingcode) [inline src/Pretzel/Commands/HungryCommand.cs]: I made it less obnoxious by describing when you should be using the command, ala parameters.

**Comment by laedit** [comment]
Perfect!

**Pull request merged**